data_store/tests/test_utils.py

```diff
@@ -0,0 +1,59 @@
+"""Tests for utils.py"""
+
+import os
+from pathlib import Path
+
+import pytest
+
+from data_store import DataStore
+
+
+class TestDataStore:
+    """Test the interface for the data sources"""
+    def setup(self):
+        cur_dir = os.path.dirname(os.path.realpath(__file__))
+        self.data_path = Path(cur_dir).joinpath("..", "..", "data").resolve()
+        self.fname = "693_UNCI.dcm"
```

Comment:
Could do in one line with pathlib and avoid `os.path`:

```python
self.data_path = Path(__file__).resolve().parent.parent.parent / "data"
```

(check the number of 'parents'... first one gives cur_dir equivalent so I think two more needed).

Reply:
Ah, didn't realise I could do that. Haven't really used pathlib much

Reply:
Done